cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/FileSetProperties.java

```diff
@@ -38,6 +38,21 @@
   public static final String DATA_EXTERNAL = "data.external";
 
   /**
+   * If true, the file set will use an existing directory as its base path and an existing Explore table.
+   * When the file set is deleted or truncated, the directory and its contents as well as the Explore table
+   * will not be deleted.
+   */
+  public static final String DATA_USE_EXISTING = "data.use.existing";
+
+  /**
+   * If true, the file set will use an existing directory as its base path and an existing Explore table,
+   * and it will take possession of them.
+   * When the file set is deleted or truncated, the directory and its contents will not be deleted
+   * exactly as if the directory had been created by this file set.
```

Comment:
yes, better. I changed it to this:

   * When the file set is deleted or truncated, the directory and its contents as well as the Explore table 
   * will be deleted, just as if they had been created by this file set.